SonarQube.TeamBuild.PreProcessor/SonarWebService.cs

```diff
@@ -43,10 +43,12 @@ public SonarWebService(IDownloader downloader, string server, ILogger logger)
 
         #region ISonarQubeServer interface
 
-        public bool TryGetQualityProfile(string projectKey, string language, out string qualityProfile)
+        public bool TryGetQualityProfile(string projectKey, string projectBranch, string language, out string qualityProfile)
         {
+            string projectId = GetProjectIdentifier(projectKey, projectBranch);
```

Comment:
Any field previously named "projectKey" should now be "projectId" if it is branch-specific.